libs/go/libp2p_node/mocks/mock_network.go

```diff
@@ -0,0 +1,191 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
In readme please explain how we can generate this.

Reply:
added reference to https://github.com/golang/mock